images/drawer_header_background_generate.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+function generate() {
+    inkscape "drawer_header_background.svg" -export-area-page --without-gui -e "../app/ui/src/main/res/drawable-$1/drawer_header_background.png" -w $2 -h $3
+}
+
+generate mdpi     384 216
+generate hdpi     576 324
+generate xhdpi    768 432
+generate xxhdpi  1152 648
+#generate xxxhdpi 1728 972
```

Comment:
Why is this commented out? Not used?

Reply:
cketty told me to support mdpi tp xxhdpi. Just in case we want to support xxxhdpi sometimes, there sizes are already here.

Reply:
Err, sounds legit. And it's not commented-out because the file generated would be big or for what reason?

Reply:
xxxhdpi is ridiculously dense and really should only be used for launcher icons because "some app launchers display your app icon as much as 25% larger than what's called for by the device's density bucket". (https://developer.android.com/training/multiscreen/screendensities)

See also https://developer.android.com/guide/topics/resources/providing-resources